pysat/tests/test_constellation.py

```diff
@@ -28,19 +28,17 @@ def test_construct_by_name(self):
         const = pysat.Constellation(name='testing')
         assert len(const.instruments) == 5
 
-    @raises(ValueError)
-    def test_construct_both(self):
+    def test_construct_both_error(self):
         """Attempt to construct a Constellation by name and list.
         Raises an error."""
-        pysat.Constellation(
-            instruments=self.instruments,
-            name='testing')
+        with pytest.raises(ValueError):
```

Comment:
I think I was wrong to originally put in the specific type of error. It may be better to check for a general error.

Reply:
Since this is an error we control, ValueError is probably the way to go.

Reply:
Yes, it's also preferable to test for the expected error message.